### DISCUSSION TITLE
Collection Tests UI

### DISCUSSION THREAD

**Discussion by brvaland** [Kong/insomnia#7620] [Ideas]
I have downloaded insomnia 9.3.1 and try to convert postman collection to insomnia. 

The conversion from postman was really smooth as it converted most of pre-request and post-response scripts including test (pm.expect > insomnia.test).

When I execute the request was expect a test tab which was missing but found test result in "timelines" section. 

Is there plans to include test tab in upcoming releases?

**Comment by subnetmarco** [comment]
Yes! We are actually working on it as we speak, I would assume it ships in the next 9.4 in 4-6 weeks.

**Reply by brvaland** [reply]
@subnetmarco Thanks for quick reply. In addition, I wanted to check if "inso cli" will execute tests on collection. The JavaScript command "insomnia.test"

Is this available in 9.3.1 or in development for 9.4+ release ?

Please advise ?

**Reply by subnetmarco** [reply]
Yes, we are also working on that. Both GUI + CLI to run tests.
